css/subject-viewer.styl

```diff
@@ -51,6 +51,7 @@
     .text-viewer
       background-color: #404040
       color: white
+      min-width: 30em
```

Comment:
The `min-width` property is causing [the text-viewer div background to overlap the task area](https://fix-text-viewer.pfe-preview.zooniverse.org/projects/scribe-2/textsubjectproject/classify?reload=0&workflow=2885) in Chrome, Edge, and Safari. This may only be an issue when the first frame is text, instead of an image. One option for a workaround is #3712.